css/booklist.css

```diff
@@ -29,8 +31,8 @@
   width: 250px;
 }
 .text_elements{
-  height: 115px;
-  overflow: hidden;
+  /* display:table-cell;
+  vertical-align: top; */
```

Comment:
This is commented-out, so there is no point adding it, right?
In that case, maybe remove the whole `.text_elements` ?
Thanks